recipes-extended/xen/files/libxl-misc-fixes.patch

```diff
@@ -0,0 +1,119 @@
+################################################################################
+SHORT DESCRIPTION:
+################################################################################
+LibXL fixes
+
+################################################################################
+LONG DESCRIPTION:
+################################################################################
+* Add some linking flexibility for building the tools out-of-tree for example
+* Allow tapdisk to be a cdrom backend, since it seems to work fine
```

Comment:
The formulation is bad, it doesn't "seem" to work fine, it does work fine :)
OpenXT today uses a loopdev for the iso, which is less than ideal. Tapdisk does it much better.
Will fix the comment